<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BioTrackerClass</class>
 <widget class="QMainWindow" name="BioTrackerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenTracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_animatonArea">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>662</width>
      <height>512</height>
     </rect>
    </property>
    <property name="title">
     <string>Tracking Area </string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="VideoView" name="videoView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>641</width>
       <height>451</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sld_video">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>641</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>2500</number>
     </property>
     <property name="pageStep">
      <number>25</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_screenshot">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>570</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="BioTracker.qrc">
      <normaloff>:/BioTracker/pix/screenshot.png</normaloff>:/BioTracker/pix/screenshot.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QPushButton" name="button_previousFrame">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>31</y>
       <width>43</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="BioTracker.qrc">
       <normaloff>:/BioTracker/pix/arrows-skip-back.png</normaloff>:/BioTracker/pix/arrows-skip-back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="button_playPause">
     <property name="geometry">
      <rect>
       <x>156</x>
       <y>31</y>
       <width>43</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="BioTracker.qrc">
       <normaloff>:/BioTracker/pix/arrow-forward1.png</normaloff>:/BioTracker/pix/arrow-forward1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="button_stop">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>31</y>
       <width>44</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="BioTracker.qrc">
       <normaloff>:/BioTracker/pix/stop.png</normaloff>:/BioTracker/pix/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="button_nextFrame">
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>31</y>
       <width>43</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="BioTracker.qrc">
       <normaloff>:/BioTracker/pix/arrows-skip-forward.png</normaloff>:/BioTracker/pix/arrows-skip-forward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="button_rewind">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>31</y>
       <width>43</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="BioTracker.qrc">
       <normaloff>:/BioTracker/pix/arrow-back.png</normaloff>:/BioTracker/pix/arrow-back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>playback speed</string>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>slow</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>92</number>
       </property>
       <property name="value">
        <number>45</number>
       </property>
       <property name="sliderPosition">
        <number>45</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>fast</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>361</width>
      <height>681</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabAlgorithm">
     <attribute name="title">
      <string>algorithm</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>201</height>
       </rect>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>choose Algorithm:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>load tracking data</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_recordTracking">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>271</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Record video with tracking info</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QLineEdit" name="edit_recordTrackingLocation">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRecordTrackingLocation">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_trackingDataOutput">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>271</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Tracking output path</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QPushButton" name="button_outputSearch">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_outputPath">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>341</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>parameters</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>341</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>tools</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Video"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoView</class>
   <extends>QGraphicsView</extends>
   <header>video/VideoView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="BioTracker.qrc"/>
 </resources>
 <connections/>
</ui>
